File: hello-kafka-api/java/src/main/java/hellokafka/Consumer.java
package hellokafka;

import org.springframework.kafka.annotation.KafkaListener;
import org.springframework.stereotype.Service;

@Service
public class Consumer {

    @KafkaListener(topics= { "hello-topic" }, groupId = "kafka_kotlin_id")
    public void consume(String message) {
        System.out.println("RECEIVE MESSAGE : " + message);
    }
}

File: hello-kafka/src/main/kotlin/hellokafka/Consumer.kt
package hellokafka

import org.springframework.kafka.annotation.KafkaListener
import org.springframework.stereotype.Service

@Service
class Consumer {

    @KafkaListener(topics= ["hello-topic"], groupId = "kafka_kotlin_id")
    fun consume(message: String) {
        println("RECEIVE MESSAGE : $message")
    }
}
